README.md

```diff
@@ -21,9 +21,11 @@ $ bundle install
 Or install it yourself:
 
 ```shell
-$ gem install uploadcare-rails -v 1.0.0
+$ gem install uploadcare-rails -v 1.1.1
 ```
 
+Uploadcare widget depends on jQuery. Rails 5.1 dropped jQuery as a default dependency so if you use rails >= 5.1 make sure that jQuery is loaded.
```

Comment:
we can use (or suggest) the widget version with bundled jquery then
https://uploadcare.com/documentation/widget/#install

Reply:
Done, see https://github.com/uploadcare/uploadcare-rails/pull/58/files#diff-04c6e90faac2675aa89e2176d2eec7d8R78